static/elements/form-field-models.js

```diff
@@ -0,0 +1,111 @@
+
+// The order of appearance of properties is the order displayed.
```

Comment:
Today I learned that order is preserved... so long as you don't also have any numeric keys.

Can we rename this file form-field-enums.js, since it has only enums?

This file starts with a blank line.

Reply:
OK, enums is fine.